go.mod

```diff
@@ -35,7 +36,7 @@ require (
 replace (
 	// Pin RHsyseng library versions
 	github.com/RHsyseng/console-cr-form => github.com/RHsyseng/console-cr-form v0.0.0-20200129200812-dc119cb0bd4d
-	github.com/RHsyseng/operator-utils => github.com/RHsyseng/operator-utils v0.0.0-20200318230845-8e48081eaa5a
+	github.com/RHsyseng/operator-utils => github.com/ruromero/operator-utils v0.0.0-20200324072727-841309f096a1
```

Comment:
needs to be changed back to `RHsyseng` ...

Reply:
oh shoot, thanks